layouts/partials/footer/footer.html

```diff
@@ -5,7 +5,12 @@
         {{ if and (.Site.Params.footer.since) (ne .Site.Params.footer.since (int (now.Format "2006"))) }}
             {{ .Site.Params.footer.since }} - 
         {{ end }}
-        {{ now.Format "2006" }} {{ .Site.Title }}
+        {{ now.Format "2006" }} 
+        {{ if .Site.Copyright }}
+            {{ .Site.Copyright }} 
+        {{ else }}
+            {{ .Site.Title }}
+        {{ end }}
```

Comment:
Thanks for the PR!

I believe it would be clearer if you use the `default` function from Hugo: https://gohugo.io/functions/default/

Then the code would be just:

```
{{ .Site.Copyright | default .Site.Title }}
```